vm/systemSmartContracts/logs.go

```diff
@@ -30,9 +30,12 @@ func (d *delegation) createAndAddLogEntryForWithdraw(
 	delegator *DelegatorData,
 	numUsers uint64,
 	wasDeleted bool,
+	withdrawFundKeys [][]byte,
 ) {
 	activeFund := d.getFundForLogEntry(delegator.ActiveFund)
-	d.createAndAddLogEntryCustom(function, address, actualUserUnBond.Bytes(), activeFund.Bytes(), big.NewInt(0).SetUint64(numUsers).Bytes(), globalFund.TotalActive.Bytes(), boolToSlice(wasDeleted))
+	topics := append([][]byte{}, actualUserUnBond.Bytes(), activeFund.Bytes(), big.NewInt(0).SetUint64(numUsers).Bytes(), globalFund.TotalActive.Bytes(), boolToSlice(wasDeleted))
```

Comment:
`topics := append(make([][]byte, 0), ........` as to avoid empty slice declaration using a literal?

Reply:
changed